.github/workflows/release-snapshot.yaml

```diff
@@ -0,0 +1,53 @@
+name: Release SNAPSHOT
+
+on:
+  push:
+    branches:
+      - master
+      - "[0-9]+"
+      - "[0-9]+.[0-9]+"
+  workflow_dispatch:
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'temurin'
+          java-version: '8'
+
+      - name: Setup Gradle
+        uses: gradle/actions/setup-gradle@v4
+
+      - name: Set version
+        id: version
+        run: |
+          VERSION=$(./gradlew :ledger:properties -q | grep "version:" | awk '{print $2}')
+          echo "version=${VERSION}" >> $GITHUB_OUTPUT
+
+      - name: Upload the SNAPSHOT versions of scalardl-ledger, scalardl-java-client-sdk, scalardl-common, and scalardl-rpc to Maven Central Repository
+        if: contains(steps.version.outputs.version, '-SNAPSHOT')
+        run: |
+          echo "${{secrets.SIGNING_SECRET_KEY_RING}}" | base64 -d > /tmp/secring.gpg
+          ./gradlew publish \
+          -Psigning.keyId="${{ secrets.SIGNING_KEY_ID }}" \
+          -P'signing.password'="${{ secrets.SIGNING_PASSWORD }}" \
+          -Psigning.secretKeyRingFile="$(echo /tmp/secring.gpg)" \
+          -PossrhUsername="${{ secrets.OSSRH_USERNAMAE }}" \
+          -PossrhPassword="${{ secrets.OSSRH_PASSWORD }}"
+
+      - name: Create SNAPSHOT container images
+        if: contains(steps.version.outputs.version, '-SNAPSHOT')
+        run: ./gradlew docker
+
+      - name: Push SNAPSHOT container images
+        if: contains(steps.version.outputs.version, '-SNAPSHOT')
+        run: |
+          docker push ghcr.io/scalar-labs/scalardl-ledger:${{ steps.version.outputs.version }}
+          docker push ghcr.io/scalar-labs/scalardl-client:${{ steps.version.outputs.version }}
```

Comment:
We will release Docker images of the community edition with the new name "scalardl-xx" while keeping the old ones until we can confirm they are no longer referred.

BTW, do we really need to release the client image? Let me know your opinion or use cases if you know.